File: News_Service_Jooq/src/main/java/com/dunice/projectNews/dto/response/PublicUserResponse.java
package com.dunice.projectNews.dto.response;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import java.util.UUID;

@Getter
@Setter
@NoArgsConstructor
public class PublicUserResponse {

    private String avatar;

    private String email;

    private String name;

    private String role;

    private UUID id;
}

File: News_Service_Kotlin/src/main/kotlin/com/dunice/advancedServiceKotlin/dto/response/PublicUserResponse.kt
package com.dunice.advancedServiceKotlin.dto.response

import java.util.*

class PublicUserResponse {
    @JvmField
    var avatar: String? = null

    @JvmField
    var email: String? = null

    @JvmField
    var name: String? = null

    @JvmField
    var role: String? = null

    @JvmField
    var id: UUID? = null
}
